data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/test_overwrite_specials_ability.cfg

```diff
@@ -0,0 +1,68 @@
+#####
+# API(s) being tested: [abilities][overwrite_specials][overwrite_filter] 
```

Comment:
My apologies - the CI failure is because of trailing whitespace on this line, and having checked back it was in the code that I gave you to cut & paste.

Reply:
Corrected, in wait to check.